tests/buyProducts_test.js

```diff
@@ -14,39 +14,48 @@ let billingDetails = {
 };
 
 const productLinks = require('../helpers/productLinksGetter');
+const checkout = require('../pages/checkout');
 let productLinks2 = productLinks.getLinks();
 console.log(productLinks2);
 
 Feature('buy product');
 
-Before(({I}) => {
+Before(async ({I, helper, homePage}) => {
     I.login(loginUser);
+    I.see('My Account');
+    homePage.clickCartIcon();
+    console.log(await homePage.checkCartIsEmpty());
+    await homePage.emptyCart();
 });
 
 Data(productLinks2).Scenario('buy product', async ({ I, productPage, checkoutPage, orderHistoryPage, current }) => {
-    I.see('My Account');
+    
     I.amOnPage(current.link);
     let price = await productPage.getProductPrice();
     console.log(price);
+    console.log(await productPage.checkColorForProduct());
```

Comment:
remove console logs. they are used for debugging mostly.

Reply:
Done.